### PR TITLE
removing broken link for writing expressions

### PR DESCRIPTION
**Pull request by henrik-ch**
Replacing with nix.dev language basics link.
(not a 100% substitute)

Trying to address  #226 but of course open for other suggestions as well.

### CODE REVIEW

File: pills/02-install-on-your-running.xml [hunk 1]
```diff
@@ -192,9 +192,9 @@
     parse="text" /></screen>
 
     <para>
-      <link
-          xlink:href="https://nixos.org/manual/nix/stable/expressions/writing-nix-expressions.html">Nix
-      expressions</link> are used to describe packages and how to
+      Nix expressions are written in the nix language  <link
+          xlink:href="https://nix.dev/tutorials/nix-language">nix 
+      language</link> and used to describe packages and how to
```
Comment (wegank) [inline pills/02-install-on-your-running.xml:197]:
```suggestion
      Nix expressions are written in the <link
          xlink:href="https://nix.dev/tutorials/nix-language">Nix 
      language</link> and used to describe packages and how to
```

**Comment by wegank** [comment]
Please also squash the commits, thanks.

**Review by wegank — APPROVED** [review]

**Comment by henrik-ch** [comment]
:exploding_head: I had some trouble with the rebase and blew up my branch, but have hopefully recreated it correctly now. @wegank sorry for this and thanks for your patience.

**Comment by kubukoz** [comment]
> commits

Could we just squash on merge?

**Pull request merged**